sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -1901,6 +1909,7 @@ case class NestedStruct(f: ClassData)
 case class DeepNestedStruct(f: NestedStruct)
 
 case class InvalidInJava(`abstract`: Int)
+private case class InvalidInJavaForNumber(`1something`: String)
```

Comment:
Ah .. this is being converted into Java class to generate Javadoc. So it causes compilation error when it does. Can you simply remove the test case here and show the reproducer to test under "How was this patch tested?"? Otherwise, seems fine.

Reply:
Nice finding!

Reply:
Should I left the test and case class commented and explain why it needs to be commented?

Reply:
You can just explain in the PR description. Let's just remove this test case, and describe how you can test in Spark shell in the PR description.

Reply:
@HyukjinKwon Done :)